### DISCUSSION TITLE
Problem statements

### DISCUSSION THREAD

**Discussion by DevanshB10** [Computer-Science-and-Engineering-GNDEC/Hackathon-1.1#35] [Q&A] [answered]
Sir, Is there only a single problem statement and are we open to choose our own problems?
@SatinderpalSingh sir.

**Comment by himagandhii** [comment]
Problem statements are provided and there are multiple options.

**Comment by SatinderpalSingh** [comment, marked as answer]
You had to send problem in the fist hour of Hackathon. You send it after one hour, that's why you have to choose from the given problems now.
